transformer_lens/components.py

```diff
@@ -1011,7 +1014,7 @@ def add_head_dimension(tensor):
         resid_mid = self.hook_resid_mid(resid_pre + attn_out)
```

Comment:
In line 995 apply DRY and move `add_head_dimension` to the top level of this file so it is defined once in the code.